apps/prairielearn/src/ee/lib/billing/components/InstructorInstanceAdminBillingForm.tsx

```diff
@@ -0,0 +1,289 @@
+            {formatEnrollmentCount(enrollmentCount, enrollmentLimit, studentBillingEnabled)}
+          </span>
+          <div
+            class={`progress flex-grow-1 ${studentBillingEnabled ? 'd-none' : ''}`}
+            style="max-width: 100px"
+          >
+            <div
+              class={`progress-bar ${enrollmentLimitProgressBarColor}`}
+              role="progressbar"
+              style={{ width: `${enrollmentLimitProgressBarPercentage}%` }}
+              aria-valuenow={enrollmentCount}
+              aria-valuemin={0}
+              aria-valuemax={enrollmentLimit}
+            ></div>
+          </div>
+        </div>
+        <div class="small text-muted">
+          {enrollmentLimitExplanation({
+            studentBillingEnabled,
+            enrollmentLimit,
+            enrollmentLimitSource,
+          })}
+        </div>
+      </div>
+
+      <div class="form-check">
+        <input
+          class="form-check-input"
+          type="checkbox"
+          name="student_billing_enabled"
+          checked={studentBillingEnabled}
+          disabled={!studentBillingCanChange}
+          value="1"
+          id="studentBillingEnabled"
+          onChange={(e) => setBasicPlanEnabled(e.currentTarget.checked)}
+        />
+        <label class="form-check-label" for="studentBillingEnabled">
+          Enable student billing for enrollments
+        </label>
+        <p class="small text-muted">
+          When student billing is enabled, students pay for access to your course instance. Enabling
+          student billing will allow your course instance to exceed any enrollment limits that would
+          otherwise apply.
+        </p>
+        {studentBillingAlert && <Alert {...studentBillingAlert} />}
+      </div>
+
+      <h2 class="h4">Features</h2>
+      <p>
+        If your course requires certain features, you can enable them so that students can pay for
+        them.
+      </p>
+
+      <div class="form-check">
+        <input
+          class="form-check-input"
+          type="checkbox"
+          name="compute_enabled"
+          checked={computeEnabled}
+          disabled={!computeCanChange}
+          value="1"
+          id="computeEnabled"
+          onChange={(e) => setComputePlanEnabled(e.currentTarget.checked)}
+        />
+        <label class="form-check-label" for="computeEnabled">
+          External grading and workspaces
+        </label>
+        <p class="small text-muted">
+          Students will be able to use questions that utilize external grading and/or workspaces.
+          This course has <strong>{pluralizeQuestionCount(externalGradingQuestionCount)}</strong>{' '}
+          that use external grading and{' '}
+          <strong>{pluralizeQuestionCount(workspaceQuestionCount)}</strong> that use workspaces.
+        </p>
+        {computeAlert && <Alert {...computeAlert} />}
+      </div>
+
+      {/* TODO: something should be showing this. But what/when? */}
```

Comment:
👀 to be addressed before merging; I need to figure this out.

Reply:
This text seems like it's related to `initialRequiredPlans`. It seems like this page is processed via morphdom in `apps/prairielearn/assets/scripts/instructorInstanceAdminBillingClient.ts`. Should we be removing that script alongside this migration?

```ts
    morphdom(
      billingForm,
      InstructorInstanceAdminBillingForm({
        ...initialProps,
        desiredRequiredPlans: requiredPlans,
      }).toString(),
    );
  });
```

Reply:
Yep, definitely; that whole script will be replaced by the Preact component.